### PR TITLE
feat(react-native): add expo video sample app

### PR DESCRIPTION
**Pull request by khushal87**
This PR focuses on adding expo video sample app for React Native SDK

### CODE REVIEW

File: sample-apps/react-native/expo-video-sample/app.json [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        "android.permission.CALL_PHONE",
+        "android.permission.ACCESS_NETWORK_STATE",
+        "android.permission.BLUETOOTH",
+        "android.permission.CAMERA",
+        "android.permission.INTERNET",
+        "android.permission.MODIFY_AUDIO_SETTINGS",
+        "android.permission.RECORD_AUDIO",
+        "android.permission.SYSTEM_ALERT_WINDOW",
+        "android.permission.WAKE_LOCK",
+        "android.permission.POST_NOTIFICATIONS"
```
Comment (santhoshvai) [inline sample-apps/react-native/expo-video-sample/app.json:39]:
```suggestion
       "android.permission.BLUETOOTH_CONNECT",
        "android.permission.FOREGROUND_SERVICE",
        "android.permission.POST_NOTIFICATIONS"
```

File: packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/02-installation/02-expo.mdx [hunk 2]
```diff
@@ -0,0 +1,83 @@
+---
+title: Installation in Expo
```
Comment (vishalnarkhede) [inline packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/02-installation/02-expo.mdx:2]:
```suggestion
title: Expo
```

File: packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/02-installation/01-react-native.mdx [hunk 3]
```diff
@@ -1,6 +1,5 @@
 ---
-title: Installation
-description: Install the SDK for video calling
+title: Installation in React Native
```
Comment (vishalnarkhede) [inline packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/02-installation/01-react-native.mdx:2]:
```suggestion
title: React Native
```

File: packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/02-installation/02-expo.mdx [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+## SDK Installation
+
+Add the SDK and its required dependencies to your project:
+
+```bash title=Terminal
+npx expo install @stream-io/video-react-native-sdk @stream-io/react-native-webrtc \
+   @config-plugins/react-native-webrtc \
+   react-native-incall-manager react-native-svg \
+   @react-native-community/netinfo @notifee/react-native
```
Comment (vishalnarkhede) [inline packages/react-native-sdk/docusaurus/docs/reactnative/01-setup/02-installation/02-expo.mdx:20]:
```suggestion
npx expo install @stream-io/video-react-native-sdk
npx expo install @stream-io/react-native-webrtc
npx expo install @config-plugins/react-native-webrtc
npx expo install react-native-incall-manager
npx expo install react-native-svg
npx expo install @react-native-community/netinfo
npx expo install @notifee/react-native
```

This is more of personal preference. Its good as is, but if you would like, I think explicit line per dependency makes it easier to read.

**Comment by vishalnarkhede** [comment]
Could you add some README to expo sample app?

- Steps to run the app
- Some screenshot or header image

**Review by santhoshvai — APPROVED** [review]

**Review by vishalnarkhede — APPROVED** [review]

**Pull request merged**